docs/source/installation.rst

```diff
@@ -150,25 +150,15 @@ Step 3: Install ``torch`` and ``torchvision`` with CPU or GPU Support
 
                     .. code::
 
-                       pip install torch==1.8.1+cu111 torchvision==0.9.1+cu111 --find-links https://download.pytorch.org/whl/torch_stable.html
+                       pip install --extra-index-url https://download.pytorch.org/whl/cu111 ivadomed
```

Comment:
I'm 90% sure that ivadomed doesn't need CUDA11. I think it should work find on CUDA10 too.

The hardware we have have on [some of our internal infra](https://intranet.neuro.polymtl.ca/computing-resources/neuropoly/gpus.html) forces us to use CUDA11, but I don't think this it's generally true. But this is something I would rather address in a follow up PR.

(I taught myself about the relationships between these computers [here](https://github.com/neuropoly/computers/issues/76)

Reply:
> I'm 90% sure that ivadomed doesn't need CUDA11. I think it should work find on CUDA10 too.

I second this. #1000 & issues raised by Naga made me realize pinning CUDA11.1 is unnecessarily making a tedious job to get started with ivadomed. This becomes significant if we think about the target audience.

> The hardware we have have on [some of our internal infra](https://intranet.neuro.polymtl.ca/computing-resources/neuropoly/gpus.html) forces us to use CUDA11, but I don't think this it's generally true. But this is something I would rather address in a follow up PR.

On point!
 
> (btw I taught myself about the relationships between the GPU/driver/CUDA/torch layers [here](https://github.com/neuropoly/computers/issues/76))

Thanks for the links... I'll definitely check them out.